tests/RequestParserLaravelTest.php

```diff
@@ -29,7 +29,7 @@ function testFilterNormalViaController()
         $request = $this->createControllerRequest($uri, $controllerClass, $query, $requestParserOptions);
 
         $requestParser = new RequestParser($request);
-        $builder = $requestParser->getBuilder();
+        $builder = $requestParser->guessModelName()->getBuilder();
```

Comment:
Lets make `guessModelName` is default, so we don't have to use it.

Reply:
Done @LIQRGV.